party/tribute/horace-goodtimes.md

```diff
@@ -6,6 +6,36 @@
  * **Combat Role:** Buffs / Healing
  * Healing spells have consequences, like requiring a sacrifice
 
+### Attributes and Skills
+
+ * **Maximum HP:** 60
```

Comment:
50 HP will be "average" for the party, I think. Clerics tend to be a little beefier so I gave you a bit more. @Roquaren will probably have more, others will have less.